docs/quickstart.rst

```diff
@@ -82,32 +88,38 @@ definitions from
 `scikit-learn <https://scikit-learn.org/stable/modules/classes.html#module-sklearn.metrics>`_
 we can evaluate metrics to get a group summary as below:
 
+.. doctest::
+
     >>> from fairlearn.metrics import group_summary
     >>> from sklearn.metrics import accuracy_score
     >>> from sklearn.tree import DecisionTreeClassifier
     >>> 
     >>> classifier = DecisionTreeClassifier(min_samples_leaf=10, max_depth=4)
     >>> classifier.fit(X, y_true)
-    >>> 
+    DecisionTreeClassifier(...)
     >>> y_pred = classifier.predict(X)
     >>> group_summary(accuracy_score, y_true, y_pred, sensitive_features=sex)
-    {'overall': 0.8445686557538159, 'by_group': {'female': 0.9256336459010306, 'male': 0.8044974759063791}}
+    {'overall': 0.844..., 'by_group': {'female': 0.925..., 'male': 0.804...}}
```

Comment:
The ellipsis is a text match, rather than a proper rounding. However, it feels wiser to do this than assume that the low level maths library never changes.